src/ServiceControlInstaller.PowerShell/ServiceControlInstaller.Powershell.dll-help.xml

```diff
@@ -938,16 +1028,18 @@ Get-SecurityIdentifier MyDomain\Foo</dev:code>
 			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>ForwardAuditMessages</maml:name>
 				<maml:description>
-					<maml:para>When enabled all messages read of the Audit queue are forwarded to queue specified AuditLog queue as part of the import into ServiceControl</maml:para>
+					<maml:para>When enabled the audit forwarding queues retains a copy of the original messages ingested by ServiceControl. The queues are not directly managed by ServiceControl and are meant as points of external integration.
+If external integration is not required, it is highly recommend to turn forwarding queues off. Otherwise, messages will accumulate unprocessed in the forwarding queue until eventually all available disk space is consumed.</maml:para>
 				</maml:description>
-				<command:parameterValue required="false" variableLength="false">Boolean</command:parameterValue>
+				<command:parameterValue required="false" variableLength="false">SwitchParameter</command:parameterValue>
 			</command:parameter>
-			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
+			<command:parameter required="false" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>ForwardErrorMessages</maml:name>
 				<maml:description>
-					<maml:para>When enabled all messages read of the Error queue are forwarded to queue specified ErrorLog queue as part of the import into ServiceControl</maml:para>
+					<maml:para>When enabled the error forwarding queues retains a copy of the original messages ingested by ServiceControl. The queues are not directly managed by ServiceControl and are meant as points of external integration.
+If external integration is not required, it is highly recommend to turn forwarding queues off. Otherwise, messages will accumulate unprocessed in the forwarding queue until eventually all available disk space is consumed.</maml:para>
```

Comment:
Should we mention here (or in docs) that it is not safe to retry a message from the Audit Log as SC is already managing those?